### PR TITLE
Churn Tests

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

File: public-node_connection_info.config [hunk 1]
```diff
@@ -0,0 +1 @@
+[
```
Comment (lionel1704) [inline public-node_connection_info.config:1]: Is this file needed?

Comment (joshuef) [reply public-node_connection_info.config:1]: no should be in gitignore, will sort that

**Review by S-Coyle — APPROVED** [review]
